test-clients/src/main/java/com/hedera/services/bdd/suites/file/negative/QueryFailuresSpec.java

```diff
@@ -67,11 +68,11 @@ private HapiApiSpec getsExpectedRejections() {
 								.hasAnswerOnlyPrecheck(INVALID_FILE_ID),
 						getFileContents("tbd")
 								.nodePayment(1_234L)
-								.hasAnswerOnlyPrecheck(FILE_DELETED)
+								.hasAnswerOnlyPrecheck(OK)
```

Comment:
Are we sending a response code `OK` even when the file is deleted ?

Reply:
Yeah, that's what it does as of now. I think we need to look into it more.

Reply:
Should we create an issue to track in future for this ?

Reply:
Yeah, I will create one or more follow up issues to track this and other issues that may show up after merging into master.